okhttp/src/main/java/io/grpc/okhttp/OkHttpClientStream.java

```diff
@@ -99,6 +101,10 @@ public int id() {
     return id;
   }
 
+  boolean useGet() {
```

Comment:
`@GuardedBy("lock")`

Reply:
Sorry for the delay in responding. This doesn't need synchronization here - the boolean can only change from false to true on the application thread, before the stream starts. Added Javadoc to clarify that the method's return value is only valid after the headers are written.